ftw/simplelayout/mapblock/profiles/default/types/ftw.simplelayout.MapBlock.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<object name="ftw.simplelayout.MapBlock"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.simplelayout" >
+
+    <!-- Basic metadata -->
+    <property name="title" i18n:translate="">MapBlock</property>
+    <property name="description" i18n:translate="">Use this block to display a map</property>
```

Comment:
And what about the description of the other blocks (in the parent package)? Shall I remove the description too?